js/queue.js

```diff
@@ -0,0 +1,39 @@
+var Queue = function () {
+  this.messageQueue = [];
+};
+
+Queue.prototype.add = function (message) {
+  this.messageQueue.push(message);
+  return this.messageQueue;
+};
+
+Queue.prototype.init = function (messages) {
+  this.messageQueue = messages;
+};
+
+Queue.prototype.getAllQueue = function () {
+  return this.messageQueue;
+};
+
+Queue.prototype.sended = function (messages) {
```

Comment:
What does this method return? p.s. the past participle of "send" is "sent"

Reply:
@squallstar that's my mistake it shouldn't return anything. This method serves only to label messages that we have sent already, so we won't send them again. I'll remove the return statement here.